src/test/java/uk/gov/dhsc/htbhf/steps/ClaimScenario.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.dhsc.htbhf.steps;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Arrays;
+
+@AllArgsConstructor
+@Getter
+public enum ClaimScenario {
+
+    IDENTITY_NOT_MATCHED("claim-identity-failed.json"),
+    ELIGIBILITY_NOT_CONFIRMED("claim-eligibility-failed.json"),
+    DUPLICATE_CLAIM("claim-duplicate.json"),
+    POSTCODE_MISMATCH("claim-postcode-mismatch.json"),
+    FULL_ADDRESS_MISMATCH("claim-full-address-mismatch.json"),
```

Comment:
Thought this was better than the rather verbose option of `BOTH_ADDRESS_LINE_ONE_AND_POSTCODE_MISMATCH`